### ISSUE TITLE
SEO boilerplate

### ISSUE THREAD

**Issue by ghost** [turtlecoin/turtlecoin.lol#17]
1. Add these to `index.html`, `brand.html`. I don't think we have any other page.

```html
<meta name="description" content="">
<meta property="og:description" content=""/>
<meta name="og:image" content=""/>
<meta property="og:site_name" content=""/>
<meta name="og:title" content="">
<meta property="og:type" content="website"/>

<meta name="twitter:card" content=""/>
<meta name="twitter:description" content=""/>
<meta name="twitter:image" content=""/>
<meta name="twitter:site" content=""/>
<meta name="twitter:title" content="">
```

2. Add: `sitemap.xml`, `robots.txt`.

3. Share, favicons

These are usually enough:
```html
<link rel="icon" type="image/png" href="" sizes="180x180">
<link rel="icon" type="image/png" href="" sizes="192x192">
<link rel="apple-touch-icon" href="" sizes="180x180">
<link rel="shortcut icon" href="xxx.ico">
```

Preview your results with the below and post them on this issue for a community review ;)

https://cards-dev.twitter.com/validator
https://developers.facebook.com/tools/debug/sharing

**Comment by ac1984** [comment]
@adj-gh pinging you as you did some of this for the brand page.

**Comment by RocksteadyTC** [comment]
What's this?

**Comment by ac1984** [comment]
@RocksteadyTC This is the minimum to be done to  get decent SEO. And for link shares to look good on Twitter and Facebook. Currently we are missing a bunch of them. See `view-source:https://turtlecoin.lol`. This ticket was created in response to comments in `#dev_marketing` and https://github.com/turtlecoin/turtlecoin.lol/pull/16#discussion_r167987656.

**Comment by Swardu** [comment]
The boilerplate is different than the current one on brand.html. Let me know what should be changed, making a PR to fix #16

**Comment by ar-x** [comment]
Done!
